.travis.yml

```diff
@@ -23,11 +23,20 @@ cache:
 
 # end to end testing template
 e2e_tests: &e2e_tests
-  stage: 'End to End tests'
+  stage: 'End to End tests CE'
+  script:
+    - STRAPI_DISABLE_EE=true yarn run -s test:generate-app -- $DB_STRING
+    - STRAPI_DISABLE_EE=true yarn run -s test:start-app & wait-on http://localhost:1337
+    - STRAPI_DISABLE_EE=true yarn run -s test:e2e
+
+# end to end testing template
+e2e_tests_ee: &e2e_tests_ee
+  stage: 'End to End tests EE'
   script:
     - yarn run -s test:generate-app -- $DB_STRING
     - yarn run -s test:start-app & wait-on http://localhost:1337
     - yarn run -s test:e2e
+  if: fork = false
```

Comment:
Will the tests run twice for EE and CE, since there is not `if: fork = true` on the stage `End to End tests CE` ?

Reply:
Yes for non fork PR we willl run both CE and EE tests